qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/UpdateCustomerView.groovy

```diff
@@ -1,139 +1,15 @@
-                "Please add the first name", 1, null)).bind(Customer.&setFirstName, Customer.&getFirstName)
-        Binding<Customer, String> bindLastName = customerBinder.forField(lastNameField).withValidator(new StringLengthValidator(
-                "Please add the last name", 1, null)).bind(Customer.&setLastName, Customer.&getLastName)
-
-        Binding<Customer, String> bindEmail = customerBinder.forField(emailField).withValidator(new EmailValidator("Given email address is not valid")).bind(Customer.&setEmail, Customer.&getEmail)
-
-        Binding<Customer, Affiliation> bindAffiliation = customerBinder.forField(affiliationComboBox).bind(Customer.&setAffiliations, Customer.&getAffiliations)
-        customerBinder.setBean(editableCustomer)
-        this.addComponent(createCustomerForm)
-    }
-
-    /**
-     * This method calls the individual Listener generation methods
-     */
-
-    private def registerListeners() {
-
-        submitButton.addClickListener({ event ->
-            submitCustomer()
-        })
-    }
-
-    /**
-     * Method which generates a listener for the submitbutton
-     *
-     * This Method checks the correctness of the user input for all fields in the affilationBinder.
-     * If the specified values are valid, the createCustomer use case is initialized, otherwise a Notification specifying the incorrect values is shown
-     */
-
-    private def submitCustomer() {
-        try {
-        if (customerBinder.writeBeanIfValid(editableCustomer)) {
-            controller.createNewCustomer(editableCustomer)
-            viewModel.successNotifications.add("Customer ${firstNameField.value} ${lastNameField.value} could be added correctly")
-        }
-        }catch(NullPointerException nullException){
-            viewModel.failureNotifications.add("Values missing for Customer ${firstNameField.value} ${lastNameField.value}")
-        }
-        catch(ValidationException validationException){
-            viewModel.failureNotifications.add("Invalid Values set for Customer ${firstNameField.value} ${lastNameField.value}")
-        }
-    }
-
-    /**
-     * Generates a Combobox, which can be used for Affiliation selection by the user
-     * and retrieves the selected Affiliation for customer creation
-     * @param affiliationList :
-     * @return Vaadin Combobox component with integrated Selectionlistener
-     */
-    private def generateAffiliationSelector(List<Affiliation> affiliationList) {
-
-        ComboBox<Affiliation> affiliationComboBox =
-                new ComboBox<>("Select an Affiliation")
-        affiliationComboBox.setPlaceholder("Select Affiliation")
-        affiliationComboBox.setItems(affiliationList)
-        affiliationComboBox.setItemCaptionGenerator({Affiliation af -> af.organisation})
-        affiliationComboBox.setEmptySelectionAllowed(false)
-
-        return affiliationComboBox
-    }
+class UpdateCustomerView {
```

Comment:
```suggestion
class UpdateCustomerView {
    //TODO implement
```
I would add the TODO here and/or throw an Exception to make it clear that this is not implemented.
```suggestion
class UpdateCustomerView {
    UpdateCustomerView() { throw new RuntimeException("Class not implemented.") }
```

Reply:
Good Idea!